plugins/connection/lxd.py

```diff
@@ -70,7 +82,13 @@ def exec_command(self, cmd, in_data=None, sudoable=True):
 
         self._display.vvv(u"EXEC {0}".format(cmd), host=self._host)
 
-        local_cmd = [self._lxc_cmd, "exec", self._host, "--", self._play_context.executable, "-c", cmd]
+        local_cmd = [
+            self._lxc_cmd,
+            "--project", self.get_option("project"),
```

Comment:
Does `--project default` always behaves the same as not being specified, for all versions of `lxc` supported by this plugin?

Reply:
Yes.